src/benchmarks/micro/Serializers/DataGenerator.cs

```diff
@@ -55,6 +55,20 @@ internal static T Generate<T>()
                 return (T)(object)CreateLargeStructWithProperties();
             if (typeof(T) == typeof(int))
                 return (T)(object)42;
+            if (typeof(T) == typeof(DateTime?))
+                return (T)(object)DateTime.UtcNow;
+            if (typeof(T) == typeof(ClassWithValueTypes))
+            {
+                return (T)(object)new ClassWithValueTypes
+                {
+                    Value = 0,
+                    NullableValueSet = 1,
+                    NullableValueNull = null,
+                    LargeStruct = CreateLargeStructWithProperties(),
+                    NullableLargeStructSet = CreateLargeStructWithProperties(),
+                    NullableLargeStructNull = null
+                };
```

Comment:
For consistency with the rest of this method, consider creating a `CreateClassWithValueTypes()` method and using that here instead of inlining the object construction.

Reply:
Updated